docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -110,17 +110,124 @@ Examples:
 |Default
 |Description
 
-|`bdd.transformer.headless.seed-relative-urls`
+4+^.^|_General_
+
+|`transformer.headless.seed-relative-urls`
 |Comma-separated list of values
 |
 |list of relative URLs, a seed URL is a URL that is fetched by the crawler to extract new URLs in it and follow them for crawling
 
-|`bdd.transformer.headless.filter-redirects`
+|`transformer.headless.filter-redirects`
 |`true` / `false`
 |`false`
 |defines whether urls that has redirect to the one that has already been included in the table are excluded from the table
+
+|`transformer.headless.socket-timeout`
+|`integer`
+|`40000`
+|socket timeout in milliseconds
+
+|`transformer.headless.connection-timeout`
+|`integer`
+|`30000`
+|connection timeout in milliseconds
+
+|`transformer.headless.max-download-size`
+|`integer`
+|`1048576`
+|max allowed size of a page in bytes. Pages larger than this size will not be fetched
```

Comment:
The formatting is strange, the first sentence starts with lower case and ends with dot, seconds starts with upper-case and does not have dot at the end